extensions/interactions/DragAndDropSort/directives/drag_and_drop_sort_response_directive.html

```diff
@@ -0,0 +1,35 @@
+<style>
+  .drag-and-drop-sort-response-content img {
+    max-height: 100px;
+  }
+  .item {
+    border: 1px solid grey;
+    width: 70px;
+    height: 70px;
+    background-color: #add8e6;
+    margin: 10px 0 0 0;
+  }
+  .subItem {
+    border: 1px solid grey;
+    width: 70px;
+    height: 70px;
+    background-color: #add8e6;
+    position: relative;
+    margin:-20px 0 0 0;
+  }
+</style>
+
+<ol ng-if="answer.length >= 1" class="drag-and-drop-sort-response-content">
+  <li ng-repeat="responses in answer track by $index">
+    <ol ng-if="responses.length >= 1">
+      <div class="item">
```

Comment:
Is it possible to collapse this and the following?

Reply:
No as the first item is logically parent and other following ones are children (subItem). They need some different CSS styling.

Reply:
Try using [`ng-class`](https://docs.angularjs.org/api/ng/directive/ngClass).

Reply:
Done